pull-primes/description.md

```diff
@@ -0,0 +1,10 @@
+
+Write a function `primes(list)`, where the list contains 4 positive integers between 0 and 169 (not included), that determines if any of the numbers are primes and prints a new list that includes only the primes (make sure that the primes are in ascending order), if there are none, print `There are no prime numbers in this list`

```

Comment:
but `primes` isn't defined in the starter? 
do they need a function that determines if a number is prime? if this solution contains two functiosn (the list filtering one and a func to determine if a number is prime), give both these function headers in the starter
I also find the problem description is alittle unclear...